blog-pagination-plan.md

```
@@ -0,0 +1,249 @@
+Use Astro's `getStaticPaths()` with `paginate()` function:
+
+```typescript
+export async function getStaticPaths({ paginate }) {
+  const posts = (await getCollection("blog")).sort(...);
+  return paginate(posts, { pageSize: 10 });
+}
+```
+
+### Component Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)